### DISCUSSION TITLE
[@typescript/cookie-session] requires new partitioned attribute

### DISCUSSION THREAD

**Discussion by omar-rahman-dev** [DefinitelyTyped/DefinitelyTyped#69119] [Issues with a @types package]
https://github.com/expressjs/cookie-session

Has recently updated to version 2.1.0 that adds partitioned attribute to support CHIPS https://developers.google.com/privacy-sandbox/3pcd/chips

The types package needs to be updated to allow cookieSession to set partitioned to true or false

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Added in #69155
